.env.example

```
@@ -0,0 +1,3 @@
+REACT_APP_MAIL_SRV
+
+REACT_APP_MAIL_API_KEY
```

API key must be on server side, not in client.
Append empty variables with `=` symbol

```suggestion
REACT_APP_MAIL_SRV=

```